moped-editor/src/routes.js

```diff
@@ -76,7 +77,7 @@ export const routes = [
         element: <DeviasStyleView />,
       },
       {
-        path: "projects/signals",
+        path: "views/signal-projects",
```

Comment:
If I'm understanding the question correctly, it will be addressed in [this enhancement](https://github.com/cityofaustin/atd-data-tech/issues/9205). @frankhereford I'm going to resolved this comment but please let me know if I haven't addressed your question!

Reply:
Ah, that makes perfect sense. Thanks for the link!